bluemira/plasma_physics/rules_of_thumb.py

```diff
@@ -130,6 +130,8 @@ def calc_cyl_safety_factor(R_0: float, A: float, B_0: float, I_p: float) -> floa
     """
     Calculate the cylindrical safety factor.
 
+    \t:math:`\\dfrac{2\\pi}{\\mu_{0}} \\dfrac{R_{0}}{A}**2 \\dfrac{B_{0}}{R_{0} I_{p}}'
```

Comment:
I think the docs is failing because the last char is " ' " not " ` " but either way this may lead to line length problems so I've put it all in a math block so you can split it over multiple lines.
```suggestion
    .. math::
        \\dfrac{2\\pi}{\\mu_{0}} \\dfrac{R_{0}}{A}**2 \\dfrac{B_{0}}{R_{0} I_{p}}
```